### ISSUE TITLE
AGV Antivirus false positive?

### ISSUE THREAD

**Issue by Slinkywinky** [ElvinC/gyroflow#29]
AVG is reporting this contains malware. It's probably a false positive.

**Comment by ElvinC** [comment]
Yep it's a false positive, someone else mentioned it as well. Any way you could check specifically what AVG is detecting?

**Comment by Slinkywinky** [comment]
Win64:Malware-Gen
